modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/events/EventHandler.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import Disposable from '../Disposable';
+
+/**
+ * EventHandler utility. It's useful for easily removing a group of
+ * listeners from different EventEmitter instances.
+ * @extends {Disposable}
+ */
+class EventHandler extends Disposable {
+
+	/**
+	 * EventHandler constructor
+	 */
+	constructor() {
+		super();
+
+		/**
+		 * An array that holds the added event handles, so the listeners can be
+		 * removed later.
+		 * @type {Array.<EventHandle>}
+		 * @protected
+		 */
+		this.eventHandles_ = [];
+	}
+
+	/**
+	 * Adds event handles to be removed later through the `removeAllListeners`
+	 * method.
+	 * @param {...(!EventHandle)} var_args
+	 */
+	add(...args) {
+		for (let i = 0; i < arguments.length; i++) {
```

Comment:
Why `...args` _and_ `arguments`? Aren't they the same thing?

Reply:
I think it was an [oversight from my old self](https://github.com/metal/metal.js/commit/0fa37ce39023dd857903f1fc5b924c80dd2d09fb#diff-3acdaa9651daa63a7d556bd548995addd526e65eab90cd8ad90ba0f20d1ab199L29)

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/489/commits/931de2ae8dee76321bf1d09ca6b3aea092322272